base/models.py

```diff
@@ -495,10 +562,10 @@ def autorizzazione_richiedi_sede_riferimento(self, richiedente, incarico, invia_
             ufficio_soci = list(sede_riferimento.delegati_ufficio_soci())
             invia_notifiche = list(invia_notifiche) + ufficio_soci
 
-        self.autorizzazione_richiedi(richiedente, (incarico, sede_riferimento), invia_notifiche=invia_notifiche, **kwargs)
+        self.autorizzazione_richiedi(richiedente, (incarico, sede_riferimento), invia_notifiche=invia_notifiche, auto=auto, **kwargs)
         return True
 
-    def autorizzazione_richiedi(self, richiedente, destinatario, invia_notifiche=None, **kwargs):
+    def autorizzazione_richiedi(self, richiedente, destinatario, invia_notifiche=None, auto=None, scadenza_giorni=None, **kwargs):
```

Comment:
https://github.com/CroceRossaItaliana/jorvik/issues/237#issuecomment-216907644

> [...] una accettazione automatica dopo un periodo di tempo anch'esso configurabile (`timedelta`), [...]

Come da requisiti, il parametro dovrebbe accettare un `timedelta` arbitrario, piuttosto che un numero intero di giorni. Questo permette, assieme all'esecuzione del cron in intervalli di poche ore, di avere scadenze molto corte (es. 6 ore per la partecipazione ad una attivita' che finisce tra 6 ore!) e scadenze molto lunghe (es. 10 anni, senza dover specificare il numero di anni).